### DISCUSSION TITLE
Cache issue? Lazy loading issue? From time to time, first click in a page, the javascript not working. I need to reload every page to make it work.

### DISCUSSION THREAD

**Discussion by ngkung** [symfony/symfony#52019] [Symfony Book: The Fast Track]
I am running my symfony project on Synology. It is version 6.0 with webpack. Most of the things are working well. 
Except that it seems to me there is cache on top.

Already no php cache on Synology WebStation.
Even tried header('Pragma: no-cache') on php.
But everytime I click a link within the project. The page quickly loaded but it seems to me it is just cache. The javascript is not working. I need to reload the page to make the script working.
Already no TWIG cache.

Any hints there? Or it is apache settings issue? Or it is symfony cache? it is still in development mode. Is this the issue?

**Comment by Dorifor** [comment]
I'm having the same issue
